front/app/lib/components/form/custom/FieldMultiSelect.tsx

```diff
@@ -54,21 +58,27 @@ export function FieldMultiSelect<T extends FormData>({
           }
           return (
             <>
+              <Text as="label" htmlFor={name} className={hideLabel ? 'sr-only' : undefined}>
+                {label}
+              </Text>
               <Select
                 {...register(name)}
                 {...rest}
-                id={field.name}
+                id={fieldId}
+                aria-labelledby={ariaLabelledBy}
+                aria-describedby={helpId}
+                aria-errormessage={errorId}
+                aria-invalid={!!errorId}
                 key={`field_multi_select_key_${JSON.stringify(options)}`}
                 isMulti={true}
```

Comment:
```suggestion
                isMulti={true}
```

ça serait bien qu'on puisse choisir si on veut du multi ou pas

Reply:
C'est ce que j'avais fait au début, mais le code n'était pas ouf